fvm/fvm.go

```diff
@@ -89,6 +89,11 @@ func (vm *VirtualMachine) GetAccount(ctx Context, address flow.Address, v state.
 // captured by the Cadence runtime and eventually disambiguated by the parent context.
 func (vm *VirtualMachine) invokeMetaTransaction(ctx Context, tx *TransactionProcedure, sth *state.StateHolder, programs *programs.Programs) (errors.Error, error) {
 	invocator := NewTransactionInvocator(zerolog.Nop())
+
+	// do not deduct fees or check storage in meta transactions
+	ctx.TransactionFeesEnabled = false
+	ctx.LimitAccountStorage = false
```

Comment:
Just to confirm these changes are on a copy of context in this scope. right? maybe adding a test to make sure context of the original transaction is intact after calling the invokemeta transaction to be safe for the future.

Reply:
that is correct, this method accepts a Context, not a reference to a Context.

I'll check if I can add that test

Reply:
Not sure how to add this test as the caller of `invokeMetaTransaction` also copies the context...

Reply:
Would it make sense to just create one Context and reuse it here? 
If not, we should rather use a `NewContextFromParent` to keep things consistent

Reply:
I think meta-transaction should always copy the context. I applied your suggestion here: https://github.com/onflow/flow-go/pull/730